src/components/reusables/reusables.module.scss

```diff
@@ -0,0 +1,21 @@
+$gray: #808080;
```

Comment:
Do we have no global file for colors?

Reply:
Global file is there, not used throughout the project except only few cases(Which should be changed). 
I've imported this from variables now.